builtin/logical/pki/path_acme_new_account.go

```diff
@@ -256,20 +274,22 @@ func (b *backend) acmeNewAccountSearchHandler(acmeCtx *acmeContext, r *logical.R
 }
 
 func (b *backend) acmeNewAccountCreateHandler(acmeCtx *acmeContext, r *logical.Request, fields *framework.FieldData, userCtx *jwsCtx, data map[string]interface{}, contact []string, termsOfServiceAgreed bool) (*logical.Response, error) {
-	if userCtx.Existing {
-		return nil, fmt.Errorf("cannot submit to newAccount with 'kid': %w", ErrMalformed)
-	}
-
 	// If the account already exists, return the existing one.
 	if b.acmeState.DoesAccountExist(acmeCtx, userCtx.Kid) {
+		// TODO: We should possibly update account values here such as contact.
```

Comment:
That's an interesting approach I never considered, neat!

Reply:
See comment below, I like this, but should go down below.

Reply:
As discussed, this was rolled back and I separated out a proper account update handler.